### PR TITLE
Add pass extension

### PR DESCRIPTION
**Pull request by benedictdudel**
With this extension you can:

- generate a new password by typing `pass generate example/password`
- edit a password file with the default editor (`$EDITOR`)
- list all available passwords by typing `pass`
- filter passwords by a search string `pass example`
- remove a password from the passwordstore
- copy a password to clipboard

Additional information to environments variables etc. can be found on:

https://www.passwordstore.org/

### CODE REVIEW

**Comment by benedictdudel** [comment]
Icon needed here too.

**Comment by ManuelSchneid3r** [comment]
> Icon needed here too.

[dialog-password](https://standards.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html#names)

**Comment by benedictdudel** [comment]
> dialog-password

Cool, didn't know about these icons!

File: Pass.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import fnmatch
+from albertv0 import *
+from shutil import which
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Pass"
+__version__ = "1.0"
+__trigger__ = "pass "
+__author__ = "Benedict Dudel"
+__dependencies__ = ["pass", "gpg", "xclip"]
```
Comment (ManuelSchneid3r) [inline Pass.py]: Remove gpg and xclip. The should have been pulled by pass.

**Comment by ManuelSchneid3r** [comment]
Please confirm the changes request when you made the changes. I cant judge if there are any issues with the code since I dont use pass, but to me it looks good. Thank you @benedictdudel for your ideas and efforts.

**Pull request merged**